bench-tps/src/bench_tps_client/tpu_client.rs

```diff
@@ -22,6 +28,14 @@ where
         self.try_send_transaction_batch(&transactions)?;
         Ok(())
     }
+    fn send_versioned_transaction_batch(
+        &self,
+        transactions: Vec<VersionedTransaction>,
+    ) -> Result<()> {
+        self.rpc_client()
+            .send_versioned_transaction_batch(transactions)?;
+        Ok(())
```

Comment:
We seem to consistently use this form elsewhere:
```suggestion
            .send_versioned_transaction_batch(transactions)
            .map_err(|err| err.into())
```